src/Std/Data/HashSet/Basic.lean

```diff
@@ -88,6 +88,13 @@ Equivalent to (but potentially faster than) calling `contains` followed by `inse
   let ⟨replaced, r⟩ := m.inner.containsThenInsertIfNew a ()
   ⟨replaced, ⟨r⟩⟩
 
+/--
+Checks if given key is contained and returns the key if it is, otherwise `none`.
+The result in the `some` case is guaranteed to be pointer equal to the key in the set.
+-/
+@[inline] def getKey? (m : HashSet α) (a : α) : Option α :=
```

Comment:
The idea is that the lemma files are structured like this pseudo-code:
```suggestion
for i in [0, num_ops):
  for j in [0, i]:
    print_lemmas_relating_op[i]_and_op[j]
```
It looks to me that this isn't the case right now, so you should reorder either the operations or the lemmas.